src/main/java/org/broadinstitute/hellbender/engine/spark/GATKRegistrator.java

```diff
@@ -34,9 +34,18 @@ public void registerClasses(Kryo kryo) {
         // fix here.
         // We are tracking this issue with (#874)
         kryo.register(Collections.unmodifiableMap(Collections.EMPTY_MAP).getClass(), new UnmodifiableCollectionsSerializer());
-
         kryo.register(Collections.unmodifiableList(Collections.EMPTY_LIST).getClass(), new UnmodifiableCollectionsSerializer());
 
+        kryo.register( Arrays.asList( "" ).getClass(), new ArraysAsListSerializer() );
+        kryo.register( Collections.EMPTY_LIST.getClass(), new CollectionsEmptyListSerializer() );
+        kryo.register( Collections.EMPTY_MAP.getClass(), new CollectionsEmptyMapSerializer() );
+        kryo.register( Collections.EMPTY_SET.getClass(), new CollectionsEmptySetSerializer() );
+        kryo.register( Collections.singletonList( "" ).getClass(), new CollectionsSingletonListSerializer() );
+        kryo.register( Collections.singleton( "" ).getClass(), new CollectionsSingletonSetSerializer() );
+        kryo.register( Collections.singletonMap( "", "" ).getClass(), new CollectionsSingletonMapSerializer() );
+        UnmodifiableCollectionsSerializer.registerSerializers( kryo );
```

Comment:
Does this make registering `collections.unmodifiableList(Collections.EMPTY_LIST) .getClass()` and friends redundant?